src/calendar.rs

```diff
@@ -60,20 +60,47 @@ pub use calendar_component::CalendarComponent;
 /// ```
 ///
 ///
-#[derive(Default, Debug, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq)]
 pub struct Calendar {
     /// Top-level calendar properties
     pub properties: Vec<Property>,
     /// Events, Todos and Venues defined in the calendar
     pub components: Vec<CalendarComponent>,
 }
 
+impl Default for Calendar {
+    fn default() -> Self {
+        Self {
+            properties: [
+                ("VERSION", "2.0").into(),
+                ("PRODID", "ICALENDAR-RS").into(),
+                ("CALSCALE", "GREGORIAN").into(),
+            ]
+            .into(),
```

Comment:
Why not use `vec!` directly?

Reply:
I wanted something a bit more neat. I added little helper instead now.